hurumap_ke/models/data.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from django.contrib.postgres.fields import ArrayField, JSONField
+from django.core.exceptions import ValidationError
+
+from wazimap.models import Geography, DBTable, FieldTable, FieldTable
+
+class DbTableChart(models.Model):
+    CHART_OPTIONS = (
+        ('column', 'Column'),
+        ('histogram', 'Histogram'),
+        ('line', 'Line'),
+        ('grouped_column', 'Grouped Column'),
+        ('pie', 'Pie Chart')
+    )
+    db_table = models.ForeignKey(DBTable, on_delete=models.CASCADE)
+    field_table = models.ForeignKey(FieldTable, on_delete=models.CASCADE)
+    chart = models.CharField(max_length=32, null=False, choices=CHART_OPTIONS)
+    fields = ArrayField(models.CharField(max_length=150, null=False, unique=True), help_text="Comma-separated fields to be included in chart. Choose one field for one Column, Histogram or Pie. And Choose two fields for grouped column")
+
+    def clean(self):
```

Comment:
I know this is something @karimkawambwa was looking into (might be for different reasons), but lets try minimizing the things we ask users. For example if I've selected column or pie as my preferred chart type (or any chart that uses a single value), there is no need to ask me for the `fields` *unless* the `FieldTable` _has more than one_ value column. At which point, you can also populate the `fields` table with available columns for me to choose from.

Reply:
I have a js that selecting a certain table shows options for fields related to that table.